<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>login</class>
 <widget class="QWidget" name="login">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>733</width>
    <height>529</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>login</string>
  </property>
  <widget class="QPushButton" name="pushButton_login">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>300</y>
     <width>61</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">				   QPushButton{
	font: 11pt &quot;楷体&quot;;
	background-color: rgb(85, 255, 127,0.3);
	
				   border-radius:10px;
				   
                                                                
                   }

                   QPushButton:pressed{
                   
	
                   }
                   
                   QPushButton:hover{
	background-color: rgb(0, 85, 255);
                   }

</string>
   </property>
   <property name="text">
    <string>登录</string>
   </property>
  </widget>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>110</y>
     <width>401</width>
     <height>111</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="label">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(170, 255, 255,0.3);
font: 11pt &quot;楷体&quot;;
border-radius:10px;</string>
      </property>
      <property name="text">
       <string> 用户名：</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="lineEdit_UserId">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true">border-radius:10px;
background-color: rgb(255, 255, 255,0.3);
font: 11pt &quot;楷体&quot;;</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_2">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(170, 255, 255,0.3);
font: 11pt &quot;楷体&quot;;
border-radius:10px;</string>
      </property>
      <property name="text">
       <string> 密码：</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="lineEdit_password">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="acceptDrops">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">border-radius:10px;
background-color: rgb(255, 255, 255,0.3);
font: 11pt &quot;楷体&quot;;</string>
      </property>
      <property name="echoMode">
       <enum>QLineEdit::Password</enum>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
